password.py

```diff
@@ -1,25 +1,60 @@
 #import
 
+#dictionaries for storing accounts,and passwords
 account_dict = {}
-sec_questions_dict = {}
+special_characters = ['!','@','#','$','%','&','*']
+i = 0
 
 def account_create():
-    print("account created")
+    print("-"*18, "\nAccount Creation!"), print("-"*18)
+    username = input("Enter a Username:\n")
+    password = input("Enter a Password:\n")
+    if len(password) < 6:
+        print("Password has to be 6 characters or more.")
+    print(security_questions())
+    security = input("Please choose a security question (Type in corresponding number):\n")
+    
 
 
+    #fail parameters
+    if len(password) < 6:
+        print("Password has to be 6 characters or more.")
+    
+    #elif password != i:
+        #while True:
+            #i+1
+            #if password == special_characters[i]:
```

Comment:
In python you can treat strings just like arrays. So you can do something like this(following is peusdocode):

```
if ['c','l', 'a'] in mystring: 
    do_something here
```
this makes it so if any of those characters appears in the string you can do something. Or for your case if none of those characters appear then you can also do something.